modules/Auth/Services/AuthService.php

```
@@ -161,13 +161,22 @@ private function sendOtpByStep($step, $identifier)
 
     private function checkOtpByStep($step, $identifier, $otp)
     {
-        if ($step->login_option == "otp" && (new Otp)->validate($identifier, $otp)->status == false) {
-            throw new \Exception(__("validation.invalid-otp"), 401);
+        if (empty($step)) {
```

The logic change here removes the exception throwing behavior. This might silently fail OTP validation without notifying the user. Consider either:
1. Keeping the exception throwing behavior
2. Adding proper error handling/logging for failed OTP validation